image/templates/helm/stackrox-secured-cluster/internal/config-shape.yaml

```diff
@@ -141,6 +142,24 @@ customize:
     podLabels: {} # dict
     podAnnotations: {} # dict
     envVars: {} # dict
+  scanner-v4-indexer:
```

Comment:
Did this work for you? The reason we use `scannerV4` instead of `scanner-v4` in other parts of the charts was because Helm templating did not like it (depends on how it's used. It's valid until it is used in a particular way. I think something like `.hi.scanner-v4-indexer.hello` is invalid in Helm. Can't recall exactly)

Reply:
This part should already be unit tested. This is part of the custom machinery for injecting pod customizations within our chart, hence the seemingly unfitting casing convention. See e.g. `admission-control`, it's the same. The referencing is established using these lines for example: https://github.com/stackrox/stackrox/blob/3ba129ae4fc4a6db84adb7f11e16df87212ca9bf/image/templates/helm/shared/templates/02-scanner-v4-07-indexer-deployment.yaml#L10